include/nlohmann/detail/macro_scope.hpp

```diff
@@ -17,6 +17,20 @@
 
 #include <nlohmann/detail/abi_macros.hpp>
 
+
+// exclude unsupported compilers
+#if !defined(JSON_SKIP_UNSUPPORTED_COMPILER_CHECK)
```

Comment:
Is this new or moved here?  If new, do we want to not allow these compilers to be used or just not support them?

Reply:
I think it was moved by 'make amalgamate' or something

Reply:
Oh, it's duplicated.  It appears again right below.

Reply:
oh yeah, I just realized that, thanks for catching.

Reply:
for some reason it is not showing up in my gitk diff!
![image](https://github.com/user-attachments/assets/90cf0ade-4987-4198-bc9a-b43df06d2b45)

Reply:
anyway sorry for the confusion, I will remove it.